doc/man3/RSA_public_encrypt.pod

```diff
@@ -46,17 +48,21 @@ Encrypting user data directly with RSA is insecure.
 
 =back
 
-B<flen> must be less than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
-based padding modes, less than RSA_size(B<rsa>) - 41 for
+B<flen> must not be more than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
+based padding modes, not more than RSA_size(B<rsa>) - 42 for
```

Comment:
Instead of a comma in this line, wouldn't an _and_ suite better.

Reply:
Ignore this.  I was wrong.

Reply:
so must not be more ... and not more ... and exactly ...